doc/api.md

```diff
@@ -406,10 +398,8 @@ As these geometry functions are often used for teaching purposes,
 visualize the normal vectors for each vertex.
 
 Lastly, to support drawing these objects with shaders,
-three functions have been added with which users can provide the addresses of the
-<!-- TODO: What are these vertex attributes? -->
-Coordinate, Normal and Texture Coordinate vertex attribs:
-<!---->
+three functions have been added with which users can provide the positions of the
+Coordinate, Normal and Texture vertex positions:
```

Comment:
No no, the other way around: "can provide the locations of the vertex position, normal, and texture coordinates attributes:"